src/Symfony/Component/HttpFoundation/File/File.php

```diff
@@ -133,4 +133,18 @@ protected function getName($name)
 
         return $originalName;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSize()
+    {
+        // Since stream wrappers and stream filters can change the file contents as read, the only
+        // reliable way to get the effective size of the file is to actually read it.
+        ob_start();
+        $size = readfile($this->getPathname());
+        ob_end_clean();
```

Comment:
This will load the full response in memory, which means will break for contents that don't fit into memory.
This totally defeat the purpose of using streams imho.
When a stream wrapper/filter is there, we'd better use chunked encoding.